rust_rocks.md

```diff
@@ -0,0 +1,13 @@
+# Rust Rocks!
+
+Consider this the unofficial Rust sales pitch.
+
+## Types
+
+Rust has strong static types. This can be very different than what you are used to if you are coming from a loosely-typed language. Don't worry, though. Once you get used to them, you'll find the types actually make your life easier. Why? Because you are making implicit assumptions explicit.
+
+TODO: Show an example that illustrates the importance of data types.
+
+## Ownership
+
+Ownership is arguably the central concept of Rust. The ownership system is __the__ reason for the safety guarantees in Rust.
```

Comment:
Yes, it's true, but this chapter without a proper explanation and examples doesn't add any value..

Reply:
Same here: "But maybe I would remove this file from the PR, factor out things that could be nice in other files as you stated with `Types` and then we discuss about a small introduction to Rust itself in another PR/discussion/issue?`

Reply:
Removed rust_rocks.md